src/api/app/controllers/webui/package_controller.rb

```diff
@@ -81,6 +81,7 @@ def show
 
     @comments = @package.comments
     @requests = []
+    @services = Service.find(project: @project.name, package: @package.name)
```

Comment:
If you only want a "flag" boolean that says if the service exists... why not to use:
```ruby
@show_trigger_services_link = Service.exists?(project: @project.name, package: @package.name)
```
This is cheaper, instead of retrieving and instantiating the whole object.

Then you have to change the view to use `@show_trigger_services_link` instead of `@services.present?`